pkg/api/dashboard.go

```diff
@@ -103,7 +103,7 @@ func GetDashboard(c *m.ReqContext) Response {
 	}
 
 	isDashboardProvisioned := &m.IsDashboardProvisionedQuery{DashboardId: dash.Id}
-	err = bus.Dispatch(isDashboardProvisioned)
+	err = bus.DispatchCtx(c.Req.Context(), isDashboardProvisioned)
```

Comment:
This is just for showing that `DispatchCtx` now can invoke both classic and ctx handlers.

Reply:
Whats the need for the old one in that case? Why not have single Dispatch function?

Reply:
@torkelo This makes it easier to transition one handler at the time.